src/main/cli/settings.c

```diff
@@ -1336,6 +1338,10 @@ const clivalue_t valueTable[] = {
     { "osd_gps_sats_show_hdop",     VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_OSD_CONFIG, offsetof(osdConfig_t, gps_sats_show_hdop) },
 #endif
 
+    { "osd_rcchannels",             VAR_UINT8  | MASTER_VALUE | MODE_ARRAY, .config.array.length = 4, PG_OSD_CONFIG, offsetof(osdConfig_t, rcChannels) },
+    { "osd_rcchannel_offsets",      VAR_INT16  | MASTER_VALUE | MODE_ARRAY, .config.array.length = 4, PG_OSD_CONFIG, offsetof(osdConfig_t, rcChannelOffsets) },
```

Comment:
Two thoughts here:
- since the intention of this is to enable users to calculate artificial telemetry values on their TX and then send / display them, they have adequate means to add scale / offset to them on the TX side, so I am not sure we need to do this as well - I think if we used a fixed conversion of the full RC range to -1000 to 1000 this should cover all use cases with some imagination on the user's side;
- if we want offset / scale, then using an actual scaling factor instead of a positive 10 based one seems to be a more flexible solution.

Reply:
@mactac
You will be the first user of this facility. What do you think?

Reply:
Is @mactac active in Github? The name doesn't pop-up when I type it...

Reply:
> @mactac
> You will be the first user of this facility. What do you think?

I think that makes sense.

Reply:
> Is @mactac active in Github? The name doesn't pop-up when I type it...

I'm here :)  Although the only thing I've contributed is this:
https://github.com/mactac/trick-on-a-switch